bin/autobahn-router/template-config.toml

```diff
@@ -77,26 +79,61 @@ dedup_queue_size = 50000
 rpc_http_url = "$RPC_HTTP_URL"
 rpc_support_compression = true
 re_snapshot_interval_secs = 1200
+request_timeout_in_seconds = 300
 
 [[sources.grpc_sources]]
 name = "router-other"
 connection_string = "$RPC_HTTP_URL_WITHOUT_TOKEN"
 token = "$RPC_TOKEN"
 retry_connection_sleep_secs = 30
 
+[[sources.quic_sources]]
+name = "quic-client"
+connection_string = "$RPC_QUIC_URL"
+retry_connection_sleep_secs = 1
+enable_gso = false
+
+[[sources]]
+region = "dfw"
+dedup_queue_size = 50000
+rpc_http_url = "$DFW_RPC_HTTP_URL"
+rpc_support_compression = true
+re_snapshot_interval_secs = 1200
+request_timeout_in_seconds = 300
+
+[[sources.grpc_sources]]
+name = "router-DFW"
+connection_string = "$DFW_RPC_HTTP_URL_WITHOUT_TOKEN"
+token = "$AMS_RPC_TOKEN"
+retry_connection_sleep_secs = 30
+
+[[sources.quic_sources]]
+name = "quic-client"
```

Comment:
minor: quic source should have region suffix as above with grpc

Reply:
fixed